### PR TITLE
fix: Some tests in minikube in version 1.18.1 is failing.

### PR DESCRIPTION
**Pull request by flacatus**
Signed-off-by: Flavius Lacatusu <flacatus@redhat.com>

### What does this PR do?
Fix minikube tests by increasing minikube version to latest and add latest templates to che-proxy job

### Screenshot/screencast of this PR

N/A
### What issues does this PR fix or reference?

Ref PR: https://github.com/eclipse-che/che-operator/pull/861

### How to test this PR?

N/A
### PR Checklist
          
[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [Custom resource definition file is up to date](https://github.com/eclipse-che/che-operator/blob/main/README.md#updating-custom-resource-definition-file)
- [ ] [Nightly OLM bundle is up to date](https://github.com/eclipse-che/che-operator/blob/main/README.md#update-nightly-olm-bundle)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: .github/workflows/minikube-backup-restore.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
       run: sudo pip install yq
     - name: Provision Minikube cluster
       run: |
-        curl -LO https://storage.googleapis.com/minikube/releases/v1.18.1/minikube-linux-amd64
+        curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
```
Comment (mmorhun) [inline .github/workflows/minikube-backup-restore.yaml:25]: I'd prefer fixed version for all tests and maybe only one with latest.

**Review by tolusha — APPROVED** [review]

**Review by mmorhun — APPROVED** [review]

**Pull request merged**